### PR TITLE
Papa 8 ADCPSL cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
new cal csvs for Papa 8 FLMA/B ADCPSLs (cruise specific - dated w/ start date of Papa 8 cruise)

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Papa 8 AM Updates [DT-65]](https://trello.com/c/NXHxU6Q7/65-papa-8-am-updates-dt-65)

**Review by asmith-whoi — APPROVED** [review]
OK

File: calibration/ADCPSL/CGINS-ADCPSL-18351__20210718.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+18351,CC_scale_factor1,0.45,constant; GP03FLMA-00008
+18351,CC_scale_factor2,0.45,constant
+18351,CC_scale_factor3,0.45,constant
+18351,CC_scale_factor4,0.45,constant
+18351,CC_bin_size,1000.0,in cm
+18351,CC_dist_first_bin,1647.0,in cm
+18351,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-18351__20210718.csv:8]: Looks good

File: calibration/ADCPSL/CGINS-ADCPSL-20502__20210718.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+20502,CC_scale_factor1,0.45,constant; GP03FLMB-00008
+20502,CC_scale_factor2,0.45,constant
+20502,CC_scale_factor3,0.45,constant
+20502,CC_scale_factor4,0.45,constant
+20502,CC_bin_size,1000.0,in cm
+20502,CC_dist_first_bin,1647.0,in cm
+20502,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-20502__20210718.csv:8]: Looks good

**Review by cdobs — APPROVED** [review]
Looks good

**Pull request merged**